packages/commons-test/src/testUtils.ts

```diff
@@ -730,3 +733,10 @@ export const getMockEServiceTemplate = (
   mode: "Deliver",
   isSignalHubEnabled: true,
 });
+
+export const getMockValidEServiceTemplateRiskAnalysis = (
+  tenantKind: TenantKind
+): EServiceTemplateRiskAnalysis => ({
+  ...getMockValidRiskAnalysis(tenantKind),
+  tenantKind,
+});
```

Comment:
Could be more clear if we move all mock methods related to Template in a separate file, like happened with `./riskAnalysisTestUtils.js`.

Reply:
I agree, this could be done in a refactor task. I would not do in this PR